src/util/profile/profile.ts

```diff
@@ -25,14 +25,15 @@ export interface UpdatableProfile {
 }
 
 export interface Profile extends UpdatableProfile {
-  readonly accessToken: Promise<TokenValueType>;
+  readonly accessToken: Promise<string>;
```

Comment:
Hmm, the getter `get accessToken()` in line 54 actually returns `TokenValueType.token` which is of type `string` so they definitely don't match in one way or the other.